src/internal/lokiutil/fake.go

```diff
@@ -42,13 +44,14 @@ func mustParseQuerystringInt64(r *http.Request, field string) int64 {
 
 func (l *FakeServer) serveConfig(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprintln(w, `limits_config:
-  max_entries_limit_per_query: 5000`)
+  max_entries_limit_per_query: 1000`)
```

Comment:
FWIW, I believe that 5,000 is the standard Loki default. If this needs to be set for a particular test, could it be set _in_ that test, so that the rest of the tests have a default config?

Reply:
I'd rather test against something more restrictive than production.